templates/logsearch-infrastructure-warden.yml

```diff
@@ -83,417 +83,417 @@ networks_default:
   subnets:
   - cloud_properties:
       name: random
-    range: 10.244.0.0/30
```

Comment:
Well we did this because it collided with IPs used by cf-warden. We did all our development in bosh-lite thats why we had to change the templates so that we can have CF, L4CF and Logsearch in the same infrastructure.